### PR TITLE
Add [b]float16 dtype and handle no commit in PT2 dashboard and 

### PR DESCRIPTION
**Pull request by huydhn**
This PR adds `[b]float16` dtype to the list together with the current `amp` default.  This also fixes a bug in which the dashboard fails to handle empty result without any commits.  

In this case, the error manifested when:
* `[b]float16` is selected in training mode where there is no data (only inference)
* Or when it is selected in main branch where, as a new feature, it's not yet available

Also piggyback on this PR a small tweak on the table height when viewing Blueberries models, i.e. https://hud.pytorch.org/benchmark/blueberries/inductor_dynamic?startTime=Wed%2C%2007%20Jun%202023%2023%3A31%3A02%20GMT&stopTime=Wed%2C%2014%20Jun%202023%2023%3A31%3A02%20GMT&granularity=hour&mode=inference&dtype=float16&lBranch=gh/desertfire/185/head&lCommit=fb55821ff816f45b8a5e8ebdeb22fdc0ec145f97&rBranch=gh/desertfire/185/head&rCommit=fb55821ff816f45b8a5e8ebdeb22fdc0ec145f97

### CODE REVIEW

**Review by desertfire — APPROVED** [review]

**Comment by desertfire** [comment]
preview link?

**Comment by huydhn** [comment]
> Visit Preview

Here you go https://torchci-git-fork-huydhn-fix-empty-commit-pt-2ea175-fbopensource.vercel.app/benchmark/compilers
